### DISCUSSION TITLE
Degradation Labelling

### DISCUSSION THREAD

**Discussion by tanjaamplo** [Amplo-GmbH/Platform-Documentation#10] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/91560861/135422540-55fb283f-cb56-45d7-8dc6-e362298b3019.png)
For me it is unclear how Degradation Labelling works. Is that something, that a Service Engineer understands? What tools do they use normally to label that?

**Comment by nielsuit227** [comment, marked as answer]
Normally they don't. We currently don't have UI for this and it's a bit tough to explain without a picture. 
However, it's actually a super simple step where they just draw a line from 100% healthy to 100% faulty over time. Let's revisit this discussion when we have UI.
